olga.yepikhina/lecture-5/login.css

```diff
@@ -0,0 +1,62 @@
+@import './index.css';
+form {
+    border: 3px solid #f1f1f1;
+}
+
+input[type=text], input[type=password] {
+    width: 100%;
+    padding: 12px 20px;
+    margin: 8px 0;
+    display: inline-block;
+    border: 1px solid darkslateblue ;
+    box-sizing: border-box;
+}
+button {
+    background-color:  darkslateblue ;
+    color: white;
+    padding: 14px 20px;
+    margin: 8px 0;
+    border: none;
+    cursor: pointer;
+    width: 100%;
+}
+
+button:hover {
+    opacity: 0.8;
+}
+
+.cancelbtn {
+    width: auto;
+    padding: 5px 18px;
+    background-color: #f44336;
+}
+
+.imgcontainer {
+    text-align: center;
+    margin: 24px 0 12px 0;
+}
+
+img.avatar {
+    width: 40%;
+    border-radius: 50%;
+}
+
+.container {
+    padding: 46px;
+    margin: 12px;
+}
+
+span.psw {
```

Comment:
psw -> better use whole word `password`.
Or `pwd` which is more often password word shortcut :)